testutil/expr.go

```diff
@@ -74,3 +78,58 @@ func ParseNamedExpr(t testing.TB, s string, name ...string) expr.Expr {
 
 	return &ne
 }
+
+var emptyEnv = expr.NewEnvironment(nil)
+
+func ExprRunner(t *testing.T, testfile string) {
+	t.Helper()
+
+	f, err := os.Open(testfile)
+	if err != nil {
+		t.Errorf("Failed to open test data, got %v (%s)", err, testfile)
+	}
+
+	ts, err := genexprtests.Parse(f, "fobar")
```

Comment:
I’m confused here, what is fobar? A typo for foobar? Shouldn’t that be `testfile`?